launchpad-guaranteed-tickets-v2/src/lib.rs

```diff
@@ -95,6 +95,11 @@ pub trait LaunchpadGuaranteedTickets:
         self.deposit_launchpad_tokens(total_tickets);
     }
 
+    #[endpoint(refundUserTickets)]
+    fn refund_user_tickets(&self, users_list: MultiValueEncoded<ManagedAddress>) {
```

Comment:
admin

Reply:
Later `add_users_to_blacklist` call seems to call `require_extended_permissions`. Should it be enough or should it be better reinforced here as well?

Reply:
Yes, it later checks the caller. I don't think we need a new check here also.